src/main/resources/lib/octane/configure.js

```diff
@@ -280,21 +288,56 @@ function octane_job_configuration(target, progress, proxy) {
                 var tdReleaseSelect = $("<td class='setting-main'>");
                 trRelease.append(tdReleaseSelect);
 
-                var select = $("<select>");
-                select.append($("<option>").text("-- Not specified --").val(-1).attr('selected', !isReleaseSpecified(pipeline)));
+                var releaseSelect = $("<select style='width: 360px;'>");
+                releaseSelect.append($("<option>").text("-- Not specified --").val(-1).attr('selected', !isReleaseSpecified(pipeline)));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['isReleaseSpecified' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/HPSoftware/hp-application-automation-tools-plugin_2/pullRequest?prid=4499562)